tests/ad_common_tests/tensor_functions_01.h

```diff
@@ -55,14 +55,16 @@ test_tensor()
   const Tensor<2, dim, ADNumberType> C5 = A * a;
   const Tensor<2, dim, ADNumberType> C6 = A / a;
 
-  const ADNumberType                 det_A       = determinant(A);
-  const ADNumberType                 tr_A        = trace(A);
-  const Tensor<2, dim, ADNumberType> A_inv       = invert(A);
-  const Tensor<2, dim, ADNumberType> A_T         = transpose(A);
-  const Tensor<2, dim, ADNumberType> A_adj       = adjugate(A);
-  const Tensor<2, dim, ADNumberType> A_cof       = cofactor(A);
-  const ADNumberType                 A_l1_norm   = l1_norm(A);
-  const ADNumberType                 A_linf_norm = linfty_norm(A);
+  const ADNumberType                 det_A = determinant(A);
+  const ADNumberType                 tr_A  = trace(A);
+  const Tensor<2, dim, ADNumberType> A_inv = invert(A);
+  const Tensor<2, dim, ADNumberType> A_T   = transpose(A);
+  const Tensor<2, dim, ADNumberType> A_adj = adjugate(A);
+  const Tensor<2, dim, ADNumberType> A_cof = cofactor(A);
+  const Tensor<2, dim, ADNumberType> A_orth =
+    project_onto_orthogonal_tensors(A, 1e-8);
```

Comment:
You are trying to test your new function here with auto-differentiation types. But that just doesn't work. Just remove this line again and write a separate test (or just go with the existing modification of the other test).

Reply:
Oke, thanks. Since I think the other test already covers the code I will just remove this line. That would mean that this function is only tested for doubles, which is fine with me. Is that what you meant?